openhtf/plugs/usb/__init__.py

```diff
@@ -118,21 +118,41 @@ def __getattr__(self, attr):
 class AdbPlug(plugs.BasePlug):
   """Plug that provides ADB."""
 
+  serial_number = None
```

Comment:
Optional - might be worth adding a bit of documentation (e.g. “Attributes” in the AdbPlug docstring) describing serial_number and mentioning that it is incompatible with conf.remote_usb/conf.ethersync